scripts/test/system.sh

```diff
@@ -24,7 +24,7 @@ function deploy_env_once() {
 function _subctl() {
     # Print GHA groups to make looking at CI output easier
     echo "::group::Running 'subctl $*'"
-    "${DAPPER_SOURCE}"/cmd/bin/subctl "$@"
+    "${DAPPER_SOURCE}"/cmd/bin/subctl "$@" || exit "$?"
```

Comment:
```suggestion
    "${DAPPER_SOURCE}"/cmd/bin/subctl "$@" || return "$?"
```

Sorry should've been `return` as it isn't running in a sub-shell